src/almanac/introduction.md

```diff
@@ -1,3 +1,72 @@
 # Welcome
 
 Welcome to the Software Gardening Almanac, an open-source handbook of applied guidance and tools for sustainable software development and maintenance.
+
+## Inspiration
+
+```{figure} assets/software-lifecycle.png
+---
+alt: A software file shown in sequences with symbols indicating creation, growth,
+  and decay.
+---
+_Software is created, grows, and decays over time._
+```
+
+Software undergoes time-based cycles phases that are not well understood or cultivated.
+Why does software grow quickly only to decay just as fast?
+How do software bugs seem to appear in the most unlikely scenarios?
+
+These cycles follow patterns from life: software is created, grows, decays, and so on (sometimes in surprising or seemingly unpredictable ways).
+Software is also connected within a complex system of relationships (similar to ecology).
+The Software Gardening Almanac posits we can understand these software lifecycle patterns and complex relationships in order to build tools which sustain or maintain their development long-term.
+
+> _"The ‘planetary garden’ is a means of considering ecology as the integration of humanity – the gardeners – into its smallest spaces._
+> _Its guiding philosophy is based on the principle of the ‘garden in motion’: do the most __for__, the minimum __against__."_
+> \- Gilles Clément
+
+The content here is inspired by ecological systems (for example, as in [_planetary gardening_](https://www.architectural-review.com/essays/in-practice/in-practice-gilles-clement-on-the-planetary-garden)).
```

Comment:
Can you be more specific about what you mean by "here"? The almanac in general? (or maybe just this introduction?

Reply:
Thanks, yes, meaning the almanac in general. I'll update for specificity here.